examples/demo/advanced/cmap_variable_sized_scatter.py

```diff
@@ -1,6 +1,5 @@
 """
 Draws a scatterplot of a set of random points of variable size and color.
- - This uses the non-standard renderer, VariableSizeScatterPlot
```

Comment:
looks like we should rename this file - it doesn't use `VariableSizeScatterPlot` anymore - just the `ColormappedScatterPlot`

Reply:
It is still an example of a scatter plot that uses variable sized markers though.  It just uses this functionality on the `ScatterMap` class directly

I'm not sure what we would rename it to

Reply:
fair enough. the name conveys behavior, not which plot type it is. never mind. let's leave it as is.